honeybee_radiance/postprocess/en17037.py

```diff
@@ -0,0 +1,264 @@
+"""Functions for post-processing EN 17037 daylight outputs."""
+import json
+import os
+
+from .annual import filter_schedule_by_hours, _process_input_folder
+
+
+def _metrics(values, occ_pattern, threshold, total_hours):
+    """Calculate annual metrics for a sensor.
+
+    Args:
+        values: Hourly illuminance values as numbers.
+        occ_pattern: A list of 0 and 1 values for hours of occupancy.
+        threshold: Threshold value for daylight autonomy.
+        total_occupied_hours: An integer for the total number of occupied hours,
+            which can be used to avoid having to sum occ pattern each time.
+
+    Returns:
+        daylight autonomy
+    """
+    def _percentage(in_v, occ_hours):
+        return round(100.0 * in_v / occ_hours, 2)
```

Comment:
Given that this is only one line of code that we use once in this module, maybe we don't need a method for it.